test/softplus_test.js

```diff
@@ -11,7 +11,7 @@ describe('test softplus', function() {
     utils.checkValue(outputTensor, expected);
   }
 
-  it.only('softplus default', function() {
+  it('softplus default', function() {
```

Comment:
Is this change part of instanceNormalization fix?

Reply:
Yes. Last I used `it.only()` for debugging **softplus** tests. 
Do you suggest adding another commit or pr for clearly fixing tests, thanks.

Reply:
adding another commit sgtm. Thanks.